src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ITupleExpression.cs

```diff
@@ -0,0 +1,473 @@
+    }
+
+    public static implicit operator short(C c)
+    {
+        return (short)c._x;
+    }
+
+    public static implicit operator string(C c)
+    {
+        return c._x.ToString();
+    }
+
+    public void M(C c1)
+    {
+        (short, string) t = /*<bind>*/(new C(0), c1)/*</bind>*/;
+        Console.WriteLine(t);
+    }
+}
+";
+            string expectedOperationTree = @"
+ITupleExpression (OperationKind.TupleExpression, Type: (System.Int16, System.String c1)) (Syntax: '(new C(0), c1)')
+  Elements(2): IConversionExpression (ConversionKind.OperatorMethod, Implicit) (OperatorMethod: System.Int16 C.op_Implicit(C c)) (OperationKind.ConversionExpression, Type: System.Int16) (Syntax: 'new C(0)')
+      IObjectCreationExpression (Constructor: C..ctor(System.Int32 x)) (OperationKind.ObjectCreationExpression, Type: C) (Syntax: 'new C(0)')
+        Arguments(1): IArgument (ArgumentKind.Explicit, Matching Parameter: x) (OperationKind.Argument) (Syntax: '0')
+            ILiteralExpression (Text: 0) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 0) (Syntax: '0')
+    IConversionExpression (ConversionKind.OperatorMethod, Implicit) (OperatorMethod: System.String C.op_Implicit(C c)) (OperationKind.ConversionExpression, Type: System.String) (Syntax: 'c1')
+      IParameterReferenceExpression: c1 (OperationKind.ParameterReferenceExpression, Type: C) (Syntax: 'c1')
+";
+            var expectedDiagnostics = DiagnosticDescription.None;
+
+            VerifyOperationTreeAndDiagnosticsForTest<TupleExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
+        }
+
+        [Fact, WorkItem(10856, "https://github.com/dotnet/roslyn/issues/10856")]
+        public void TupleExpression_UserDefinedConversionFromTupleExpression()
+        {
+            string source = @"
+using System;
+
+class C
+{
+    private readonly int _x;
+    public C(int x)
+    {
+        _x = x;
+    }
+
+    public static implicit operator C((int, string) x)
+    {
+        return new C(x.Item1);
+    }
+
+    public static implicit operator (int, string)(C c)
+    {
+        return (c._x, c._x.ToString());
+    }
+
+    public void M(C c1)
+    {
+        C t /*<bind>*/= (0, null)/*</bind>*/;
+        Console.WriteLine(t);
+    }
+}
+";
+            string expectedOperationTree = @"
+IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclarationStatement) (Syntax: 'C t /*<bind ... *</bind>*/;')
+  IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration) (Syntax: 'C t /*<bind ... *</bind>*/;')
+    Variables: Local_1: C t
+    Initializer: IConversionExpression (ConversionKind.OperatorMethod, Implicit) (OperatorMethod: C C.op_Implicit((System.Int32, System.String) x)) (OperationKind.ConversionExpression, Type: C) (Syntax: '(0, null)')
+        IConversionExpression (ConversionKind.CSharp, Implicit) (OperationKind.ConversionExpression, Type: (System.Int32, System.String)) (Syntax: '(0, null)')
```

Comment:
> Is this a compiler issue?

We should not expect that the shape of the bound tree is what we want IOperation tree to have. It is possible that this particular problem can be addressed by adjusting the shape of the bound tree, it has to be investigated. I would treet this an IOperation issue as long as compiler behaves as expected (the shape of the bound tree is not part of the behavior) and other public APIs behave as expected too.